static/src/javascripts/bootstraps/commercial.js

```diff
@@ -97,17 +100,21 @@ define([
 
     function loadModules(modules, baseline) {
 
-        ophanTracking.addStartTimeBaseline(baseline);
+        performanceLogging.addStartTimeBaseline(baseline);
 
         var modulePromises = [];
 
         modules.forEach(function (pair) {
 
             var moduleName = pair[0];
+            var moduleInit = pair[1];
+            var hasCustomTiming = pair[2];
 
             robust.catchErrorsAndLog(moduleName, function () {
-                var modulePromise = pair[1]().then(function(){
-                    ophanTracking.moduleCheckpoint(moduleName, baseline);
+                var modulePromise = moduleInit(moduleName).then(function(){
```

Comment:
The only thing I could think of was moving the name each module into the module itself, but it doesn't really help, and it might even be harder to see.